### PR TITLE
Update extension node doc w/ tag

### PR DESCRIPTION
**Pull request by zkayyali812**
Ensures tag is added to identify extension node instances as being associated with their related cloudformation stack.

### CODE REVIEW

File: stories/topics/proc-aap-aws-add-extension-nodes.adoc [hunk 1]
```diff
@@ -62,7 +62,11 @@ The default is m5.large.
 . In the *Configure advanced options* page, click btn:[Next].
 . Configure the *Group size* capacity, then click btn:[Next].
 . In the *Add notifications* page, click btn:[Next].
-. In the *Add tags* page, click btn:[Next].
+. In the *Add tags* page add the following tag:
+.. Key: `aap:cloudformation:main-stack-name` Value: `<CLOUDFORMATION_STACK_NAME>`
+.. Replace `<CLOUDFORMATION_STACK_NAME>` with the name of your cloudformation stack
+.. Ensure `Tag new instances` is checked
```
Comment (ianf77) [inline stories/topics/proc-aap-aws-add-extension-nodes.adoc:68]: Full stop at end.

File: stories/topics/proc-aap-aws-add-extension-nodes.adoc [hunk 2]
```diff
@@ -62,7 +62,11 @@ The default is m5.large.
 . In the *Configure advanced options* page, click btn:[Next].
 . Configure the *Group size* capacity, then click btn:[Next].
 . In the *Add notifications* page, click btn:[Next].
-. In the *Add tags* page, click btn:[Next].
+. In the *Add tags* page add the following tag:
+.. Key: `aap:cloudformation:main-stack-name` Value: `<CLOUDFORMATION_STACK_NAME>`
+.. Replace `<CLOUDFORMATION_STACK_NAME>` with the name of your cloudformation stack
```
Comment (ianf77) [inline stories/topics/proc-aap-aws-add-extension-nodes.adoc:67]: Full stop at end.

**Review by ianf77 — COMMENTED** [review]
A few comments.

File: stories/topics/proc-aap-aws-add-extension-nodes.adoc [hunk 3]
```diff
@@ -64,8 +64,8 @@ The default is m5.large.
 . In the *Add notifications* page, click btn:[Next].
 . In the *Add tags* page add the following tag:
 .. Key: `aap:cloudformation:main-stack-name` Value: `<CLOUDFORMATION_STACK_NAME>`
-.. Replace `<CLOUDFORMATION_STACK_NAME>` with the name of your cloudformation stack
-.. Ensure `Tag new instances` is checked
+.. Replace `<CLOUDFORMATION_STACK_NAME>` with the name of your cloudformation stack.
+.. Ensure `Tag new instances` is checked.
 . click btn:[Next].
```
Comment (ianf77) [inline stories/topics/proc-aap-aws-add-extension-nodes.adoc:69]: click should have a capital letter.

**Review by ianf77 — COMMENTED** [review]
Another comment.

File: stories/topics/proc-aap-aws-add-extension-nodes.adoc (hunk 3 above)
Comment (zkayyali812) [reply stories/topics/proc-aap-aws-add-extension-nodes.adoc]: Updated!

**Review by ianf77 — APPROVED** [review]

**Pull request merged**